G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.close/ShouldCloseSheet_WhenCloseingSheet.G1ANT

```diff
@@ -0,0 +1,8 @@
+﻿addon googledocs version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥FileID = ‴147EH2vEjGVtbzzkT6XaI0eNZlY5Ec91wlvxN3HC4GMc‴
```

Comment:
What is this magic value? If it is an id of existing google document please add a comment.

Reply:
Its taken from our nunit tests